### PR TITLE
selenium: fix checkver

### PR DESCRIPTION
**Pull request by linsui**
https://github.com/ScoopInstaller/Main/issues/150

Is java wrapper needed?

### CODE REVIEW

File: bucket/selenium.json [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+    "suggest": {
+        "JDK": [
+            "java/openjdk",
+            "java/oraclejdk"
+        ]
+    },
     "checkver": {
-        "url": "https://www.seleniumhq.org/download/",
-        "re": "Download version\\s+<a href=\".*\">(?<version>(?<short>\\d+\\.\\d+).\\d+)</"
+        "github": "https://github.com/SeleniumHQ/selenium",
```
Comment (niheaven) [inline bucket/selenium.json]:
```suggestion
        "url": "https://selenium.dev/downloads/",
```

File: bucket/selenium.json [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+        "regex": "Selenium ([\\d.]+)"
```
Comment (niheaven) [inline bucket/selenium.json]:
```suggestion
        "regex": "Latest stable version.*?>([\\d.]+)<"
```

File: bucket/selenium.json [hunk 3]
```diff
@@ -1,16 +1,22 @@
 {
     "version": "3.141.59",
-    "homepage": "https://www.seleniumhq.org/",
-    "description": "A portable framework for testing web applications.",
+    "homepage": "https://selenium.dev",
+    "description": "A smart proxy server that allows Selenium tests to route commands to remote web browser instances",
```
Comment (niheaven) [inline bucket/selenium.json]:
```suggestion
    "description": "A browser automation framework and ecosystem",
```

See https://github.com/SeleniumHQ/selenium

**Comment by niheaven** [comment]
Java wrapper is not needed, it will be created in scoop core.

File: bucket/selenium.json (hunk 3 above)
Comment (linsui) [reply bucket/selenium.json]: This manifest is only for Selenium Server (Grid).  The description in github repo is for all the Selenium components.

**Pull request merged**